classes/class-p4-search.php

```diff
@@ -209,14 +209,6 @@ protected function set_general_context( &$context ) {
 			$context['selected_sort'] = $this->selected_sort;
 			$context['filters']       = $this->filters;
 			$context['found_posts']   = count( (array) $this->all_posts );
-
-			// Footer context.
-			$context['footer_social_menu']    = wp_get_nav_menu_items( 'Footer Social' );
-			$context['footer_primary_menu']   = wp_get_nav_menu_items( 'Footer Primary' );
-			$context['footer_secondary_menu'] = wp_get_nav_menu_items( 'Footer Secondary' );
-			$context['copyright_text']        = planet4_get_option( 'copyright' ) ?? '';
-			$context['page_category']         = $category->name ?? __( 'Search page', 'planet4-master-theme' );
```

Comment:
Page category is not needed or it was deleted by mistake?

Reply:
Yes it was deleted by mistake. I corrected it by creating another PR #162